tests/test_all_rules.py

```diff
@@ -436,6 +436,40 @@ def test_no_duplicate_tags(self):
         if invalid:
             self.fail(f'Rules with duplicate tags:\n{invalid}')
 
+    def test_wildcard_tag_validation(self):
```

Comment:
Usually these test cases iterate over all rules. This test case may be better moved elsewhere.